automatic/chromium/update.ps1

```diff
@@ -1,74 +1,74 @@
+Import-Module au
 
-import-module au
-    $releases = 'https://chromium.woolyss.com/api/v5/?os=win<bit>&type=<type>&out=json'
-    $ChecksumType = 'sha256'
+$checksumType = 'sha256'
 
 function global:au_SearchReplace {
   @{
     ".\legal\verification.txt" = @{
-    "(?i)(\s*32\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL32)>"
-    "(?i)(\s*64\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL64)>"
-    "(?i)(^\s*checksum\s*type\:).*"            = "`${1} $($Latest.ChecksumType32)"
-    "(?i)(^\s*checksum32\:).*"                 = "`${1} $($Latest.Checksum32)"
-    "(?i)(^\s*checksum64\:).*"                 = "`${1} $($Latest.Checksum64)"
+      "(?i)(\s*32\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL32)>"
+      "(?i)(\s*64\-Bit Software.*)\<.*\>"        = "`${1}<$($Latest.URL64)>"
+      "(?i)(^\s*checksum\s*type\:).*"            = "`${1} $($Latest.ChecksumType32)"
+      "(?i)(^\s*checksum32\:).*"                 = "`${1} $($Latest.Checksum32)"
+      "(?i)(^\s*checksum64\:).*"                 = "`${1} $($Latest.Checksum64)"
     }
     ".\tools\chocolateyInstall.ps1" = @{
-    '(^[$]version\s*=\s*)(".*")'               = "`$1""$($Latest.Version)"""
-	  "(?i)(^\s*file\s*=\s*`"[$]toolsdir\\).*"   = "`${1}$($Latest.FileName32)`""
-	  "(?i)(^\s*file64\s*=\s*`"[$]toolsdir\\).*" = "`${1}$($Latest.FileName64)`""
+      '(^[$]version\s*=\s*)(["''].*["''])'       = "`$1'$($Latest.Version)'"
+      "(?i)(^\s*url\s*=\s*').*"                  = "`${1}$($Latest.URL32)'"
+      "(?i)(^\s*file64\s*=\s*`"[$]toolsdir\\).*" = "`${1}$($Latest.FileName64)`""
     }
     ".\chromium.nuspec" = @{
-    "(?i)(^\s*\<title\>).*(\<\/title\>)"       = "`${1}$($Latest.Title)`${2}"
+      "(?i)(^\s*\<title\>).*(\<\/title\>)"       = "`${1}$($Latest.Title)`${2}"
     }
   }
 }
 
 function global:au_BeforeUpdate {
     Get-RemoteFiles -Purge -FileNameBase "$($Latest.PackageName)"
+
+    # This should be reworked when Get-RemoteFiles can get _only_ x64 URLs
+    Remove-Item $PSScriptRoot\tools\chromium_x32.exe
```

Comment:
I think this is fine, otherwise you would need to do more handling even if `Get-RemoteFiles` only handled x64 URLs (you would still need to download and calculate the checksum for x32).

As such, the comment can be removed.